kaggle_environments/envs/connectx/connectx.json

```diff
@@ -51,7 +51,8 @@
       "defaults": [1, 2],
       "description": "Which checkers are the agents.",
       "enum": [1, 2]
-    }
+    },
+    "remainingOverageTime": 60
```

Comment:
nit: The word `remaining` feels weird in a config value, maybe something like `timeoutOverageAllowance`, though that's wordy and renames aren't fun.

Reply:
Not sure if this changes things for you, but this is not a config value it's an observation value since it changes over time. My goal in including the `remaining` term was to make it clear that it was a dynamic field that would change from step to step, but I'm definitely open to other names if you have ideas.

Reply:
👍 Yeah that does sound better in that context, it does still feel like a config file, but you're right that the observation portion is intended to change over time.